lcx-api/test/adapter_test.js

```diff
@@ -0,0 +1,50 @@
+const assert = require('chai').assert
+const createRequest = require('../adapter').createRequest
+
+describe('createRequest', () => {
+  const jobID = '1'
+
+  context('successful calls', () => {
```

Comment:
Please tag all tests that require communication with the real backend and presumably an `API_KEY` with `@integration`:

```suggestion
  context('successful calls @integration', () => {
```